distribution/docs/src/main/resources/content/_architectures/_securityFramework/realms.adoc

```diff
@@ -24,9 +25,14 @@ A Subject is an application user and all available security-relevant information
 |Classname
 |Description
 
-|ddf.security.realm.sts.StsRealm
-|This realm delegates authentication to the Secure Token Service (STS). It creates a `RequestSecurityToken` message from the incoming Authentication Token and converts a successful STS response into a Subject.
+|org.codice.ddf.security.guest.realm.GuestRealm
+|This realm checks if Guest access is allowed on the incoming Authentication Token, and if so the Guest realm returns the Guest Authentication Info.
+
+|org.codice.ddf.security.oidc.realm.OidcRealm
+|This realm takes in any Oidc/OAuth credentials found on the incoming Authentication Token, and if so does its best to resolve an ID_Token using said credentials. The ID_Token is then used to put together the resulting Authentication Info.
```

Comment:
Rather than "does its best" lets be a little more exact about this. Something like: "This realm takes in any OIDC/OAuth credentials found on the incoming Authentication Token, and if so, resolves an ID_Token using those credentials."